.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -16,9 +15,9 @@ assignees: ''
 -->
 
 <!-- Checked checkbox should look like this: [x] -->
- - [ ] I am on the [latest](https://github.com/ActivityWatch/activitywatch/releases/latest) ActivityWatch version.
- - [ ] I have searched the issues of this repo and believe that this is not a duplicate.
 
+- [ ] I am on the [latest](https://github.com/proximity-tech/activitywatch/releases/latest) ActivityWatch version.
```

Comment:
I'm a Github noob but, 
Redirecting the all the links in the official Readme to your fork seems either extremely sleasy or downright malcious. 🤔